api/test/logs/logger_test.cc

```diff
@@ -70,17 +103,14 @@ class TestProvider : public LoggerProvider
 
 TEST(Logger, PushLoggerImplementation)
 {
-  // Push the new loggerprovider class into the global singleton
+  // Push the new loggerprovider class into the API
   auto test_provider = shared_ptr<LoggerProvider>(new TestProvider());
   Provider::SetLoggerProvider(test_provider);
 
   auto lp = Provider::GetLoggerProvider();
 
-  // GetLogger(name, options) function
+  // Get a logger instance and check whether it's GetName() method returns
+  // "test logger" as defined in the custom implementation
   auto logger = lp->GetLogger("TestLogger");
-
-  // GetLogger(name, args) function
-  std::array<string_view, 1> sv{"string"};
```

Comment:
no harm in leaving this in? needed for code cov

Reply:
Yup it's still in but the GitHub diff tool said that it was taken out and readded somewhere else... confusing